### DISCUSSION TITLE
Scripting: How to extract MQTT JSON Array Element?

### DISCUSSION THREAD

**Discussion by jkwim** [arendst/Tasmota#21962] [Q&A]
I have the following MQTT message and want to extract the Array Elements from "Energy:"
`
tele/tasmota/SENSOR = {"Time":"2024-08-14T19:46:28","LD2410":{"Distance":[74.0,197.0,115.0],"Energy":[100,80]}}
`
Tried following but not successful:

```
>T
move=LD2410#Energy#2

```

**Comment by jkwim** [comment]
Figured it out

```
>E
print E:
move=LD2410#Energy[1]
if upd[move]==1 {
 print E: move %move%
}

```

**Comment by HofeBY** [comment]
try without .

.>T
move=LD2410#Energy

.>W
move = %2move% whatever
